examples/rondpoint/README.md

```diff
@@ -0,0 +1,38 @@
+# Example uniffi component: "Rondpoint"
+
+This example is to exercise complex data types through roundtripping.
+
+* [`./src/rondpoint.idl`](./src/rondpoint.idl), the component interface definition which defines the main object and its methods. This is processed by functions in [`./build.rs`](./build.rs)
+  to generate Rust scaffolding for the component.
+* [`./src/lib.rs`](./src/lib.rs), the core implementation of the component in Rust. This basically
+  pulls in the generated Rust scaffolding via `include!()` and fills in function implementations.
+* [`./src/main.rs`](./src/main.rs) generates a helper executable that can be used for working with
+  foreign language bindings, while guaranteeing that those bindings use the same version of `uniffi`
+  as the compiled component.
+* Some small test scripts that double as API examples in each target foreign language:
+  * Kotlin [`./tests/bindings/test_rondpoint.kts`](./tests/bindings/test_rondpoint.kts)
+  * Swift [`./tests/bindings/test_rondpoint.swift`](./tests/bindings/test_rondpoint.swift)
+  * Python [`./tests/bindings/test_rondpoint.py`](./tests/bindings/test_rondpoint.py)
+
+If you want to try it out, you will need:
+
+* The [Kotlin command-line tools](https://kotlinlang.org/docs/tutorials/command-line.html), particularly `kotlinc`.
+* The [Java Native Access](https://github.com/java-native-access/jna#download) JAR downloaded and its path
+  added to your `$CLASSPATH` environment variable.
+* Python 3
+* The [Swift command-line tools](https://swift.org/download/), particularly `swift`, `swiftc` and
+  the `Foundation` package.
```

Comment:
As a meta-level comment, we can probably figure out how to consolidate these detailed how-to docs from the examples in a single place, rather than copy-pasting them for each one. Not for this PR though.